File: bronze4/31612.cpp
#include <iostream>
#include <vector>

int main(void) 
{
    int N = 0;
    std::cin >> N;

    std::vector<char> S(N);

    for (int i = 0; i < N; i++) 
    {
        std::cin >> S[i];
    }

    int sum = 0;
    for (char c : S) {
        if (c == 'j' || c == 'i') 
        sum += 2;
        else if (c == 'o') 
        sum += 1;
    }

    std::cout << sum << std::endl;

    return 0;
}

File: bronze4/31612.js
const input = require('fs').readFileSync('/dev/stdin', 'utf8').trim().split('\n');

const N = Number(input[0]);
const S = input[1];

let sum = 0;
for (let i = 0; i < N; i++) {
    if (S[i] === 'j' || S[i] === 'i') 
      sum += 2;
    else if (S[i] === 'o') 
      sum += 1;
}

console.log(sum);
